### DISCUSSION TITLE
Pushing a branch

### DISCUSSION THREAD

**Discussion by GustavHaapalahti** [lensfun/lensfun#1370] [Q&A] [answered]
Hello,

I am trying to make a contribution by pushing a branch with new lens data, but I get "ERROR: Permission to lensfun/lensfun.git denied".
Do I need special write access to contribute via git?

Best Regards Gustav

**Comment by GustavHaapalahti** [comment, marked as answer]
I found out that I could fork the repository, push my changes to the fork and make a pull request from a branch there.
First time github user :-)

**Reply by Sturmflut** [reply]
I'll have a look at #1380 and #1403 next. Will be a couple of days, though.
